src/Ares.php

```diff
@@ -1,46 +1,85 @@
 <?php
+/**
+ * This file is part of Ares.
+ * Copyright (c) 2015 Václav Vrbka (http://aurielle.cz)
+ */
 
-namespace Aurielle\Ares;
+namespace Grifart\Ares;
 
 use Nette;
 
+
 /**
  * @author Milan Matějček <milan.matejcek@gmail.com>
- * @author Václav Vrbka <aurielle@aurielle.cz>
+ * @author Václav Vrbka <vaclav.vrbka@grifart.cz>
  */
 class Ares extends Nette\Object
 {
+	/** @var HttpDriver */
+	private $httpDriver;
 
-	/** @var IRequest */
-	private $request;
+	/** @var Cache */
+	private $cache;
 
-	public function __construct(IRequest $request = NULL)
+
+	public function __construct(HttpDriver $httpDriver, Cache $cache)

```

Comment:
`public function __construct(IDriver $driver, ICacheStorage $cacheStorage);`
- umožní vyměňovat driver (tu jde jen o extrakci interface z třídy)
- Nette\Caching\Cache je už kokrétní namespace. Nevím jak efektivně se dá takový objekt vytvořit v kontejneru (bez továrny). Jsem tedy pro předávání ICacheStorage u kterého je vše jasnější. Jen si budeme muset vymyslet název pro Cache namespace. http://api.nette.org/2.3.4/Nette.Caching.Cache.html#___construct